fdbserver/ClusterController.actor.cpp

```diff
@@ -1567,12 +1570,22 @@ class ClusterControllerData {
 			return false;
 		}
 
+		if (oldTLogFit.count + oldInFit.proxy.count + oldInFit.grvProxy.count + oldInFit.resolver.count >
```

Comment:
I see you documented this function at the top level. Since it's a pretty substantial function, we could also consider adding a comment on this particular block to explain what it's being put here for. I'll leave that decision to you.

Reply:
done